### ISSUE TITLE
Microdata attributes should be allowed on non-itemprop <meta> elements in the <head>

### ISSUE THREAD

**Issue by unor** [validator/validator#471]
Test documents:

```html
<!DOCTYPE html>
<meta name="keywords" content="test" itemscope>
<title>Test</title>
```

```html
<!DOCTYPE html>
<meta http-equiv="default-style" content="test" itemscope>
<title>Test</title>
```

```html
<!DOCTYPE html>
<meta charset="utf-8" itemscope>
<title>Test</title>
```

https://validator.w3.org/nu/ and https://checker.html5.org/ give this error:

> Attribute `itemscope` not allowed on element `meta` at this point.

The error message lists which attributes are allowed. The list includes the link to the "global attributes", which ([in the WHATWG version](https://html.spec.whatwg.org/multipage/dom.html#global-attributes)) contain `itemscope`.

Am I missing something or should the test documents be valid?

It does work with `link`-`rel` elements:

```html
<!DOCTYPE html>
<link rel="alternate" href="/test" itemscope>
<title>Test</title>
```

(I’m coming from https://github.com/schemaorg/schemaorg/issues/1540#issuecomment-283820674.)

**Comment by jvandriel** [comment]
But what about using the <template> element as a 'wrapper', eg. the element with the @itemscope?

**Comment by sideshowbarker** [comment]
Thanks for taking time to report this. The examples in the issue description are conforming per the HTML spec. The  only special restriction in the spec on use of the global Microdata attributes on `meta[name}` is the one for `itemprop`:

> [Exactly one of the `name`, `http-equiv`, `charset`, and `itemprop` attributes must be specified.](https://html.spec.whatwg.org/#the-meta-element:names:-the-itemprop-attribute-5)

So I made a fix and pushed it to https://checker.html5.org/ and https://validator.w3.org/nu/

**Comment by sideshowbarker** [comment]
> But what about using the ```<template>``` element as a 'wrapper', eg. the element with the @itemscope - Yes, No?

I don’t understand why you’d do that but personally have no opinion on it. It’s conforming at least. Anything’s allowed in `template`. But of course if you put something into `template` in your source markup it’s not going to end up exposed in the DOM after parsing. I have no idea whether that makes any difference for schema.org usage but I would think it certainly makes a difference for pretty much any other use case.
